### DISCUSSION TITLE
2024/12/29/2024%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#9] [Announcements]
# 2024/12/29/2024%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

2024 年，有了一些不一样的。

https://blog.byteloid.one/2024/12/29/2024%E5%B9%B4%E5%BA%A6%E6%80%BB%E7%BB%93/

**Comment by Seayay** [comment]
一年 16 篇，你好强大

**Comment by liuzhen9320** [comment]
新的一年共同进步！

**Comment by dongzhimin-xz** [comment]
😯求教横幅上的特效怎么实现🥺🥹
